lib/tori/backend/s3.rb

```diff
@@ -6,15 +6,35 @@ module Backend
     class S3
       DEFAULT_CONTENT_TYPE = 'text/plain'.freeze
       attr_accessor :bucket
+      attr_reader   :client
+
       # Must be set bucket name.
       #   And it use aws-sdk-core >= 2.0
-      #   ENV["TORI_ACCESS_KEY"] > aws-sdk credentials
+      #   ENV takes precedence over credentials file and instance profile
       #
       # example:
       #   Tori.config.backend = Tori::Backend::S3.new(bucket: 'tori_bucket')
-      def initialize(bucket:)
-        @bucket = bucket
-        @client = nil
+      def initialize(options = {})
+        @bucket = options[:bucket] || (fail ArgumentError, 'missing keyword: bucket')
+
+        region =
+          options[:region] || ENV['TORI_AWS_REGION'] || ENV['AWS_REGION'] || Aws.config[:region]
+        @client =
+          case
+          when options[:access_key_id] && option[:secret_access_key]
+            Aws::S3::Client.new(
+              access_key_id:     options[:access_key_id],
+              secret_access_key: options[:secret_access_key],
+              region:            region)
+          when ENV['TORI_AWS_ACCESS_KEY_ID'] && ENV[:TORI_AWS_SECRET_ACCESS_KEY]
```

Comment:
This is `ENV['TORI_AWS_SECRET_ACCESS_KEY']` dont you ?
This code raise an error `TypeError: no implicit conversion of Symbol into String`